test/util.ts

```diff
@@ -697,7 +697,9 @@ describe('common/util', () => {
   describe('makeAuthenticatedRequestFactory', () => {
     const authClient = {
       getCredentials() {},
-      _cachedProjectId: 'project-id',
+      getProjectId: () => {
```

Comment:
Since this is async in the library, should it be an async function here?

Reply:
Makes sense to me! Changed 👍🏻